src/statmodels.jl

```diff
@@ -160,17 +160,50 @@ df_residual(obj::RegressionModel) = error("df_residual is not defined for $(type
 ## Nms are the coefficient names, corresponding to rows in the table
 type CoefTable
     mat::Matrix
+    cols::Tuple
```

Comment:
It is easy to create a tuple by `(v1, v2, ...)` when calling the function `CoefTable`. 
But it is not so easy to create a vector of vectors.